src/index.js

```diff
@@ -80,7 +80,7 @@ function ParseServer({
   cloud,
   collectionPrefix = '',
   clientKey = '',
-  javascriptKey = randomString(20),
+  javascriptKey = '',
```

Comment:
it's still set to '' we don't check for key lengths. I'll test that in the test repo